stix_shifter/stix_transmission/stix_transmission.py

```diff
@@ -110,5 +68,43 @@ def is_async(self):
             return self.entry_point.is_async()
         except Exception as ex:
             return_obj = dict()
-            ErrorResponder.fill_error(return_obj, error=ex, connector=self.connector)
+            ErrorResponder.fill_error(return_obj, error=ex)
```

Comment:
why is `connector=self.connector` removed? is it not accessible anymore? this arg basically adds the connector name in the error response. You may have to verify that if it still prints the connector name.